docs/reference/metricbeat/metricbeat-module-elasticsearch.md

```diff
@@ -85,8 +85,15 @@ metricbeat.modules:
   #scope: node
 ```
 
-This module supports TLS connections when using `ssl` config field, as described in [SSL](/reference/metricbeat/configuration-ssl.md). It also supports the options described in [Standard HTTP config options](/reference/metricbeat/configuration-metricbeat.md#module-http-config-options).
+You cannot specify the `username`/`password` settings _and_ `api_key` at the same time.
+
+When used, the `api_key`  configuration can be specified as:
 
+* {applies_to}`stack: ga 9.2.0` The unencoded `id:api_key` format (`api_key: "foo:bar"`) or the Base64-encoded `id:api_key` format (`api_key: "Zm9vOmJhcgo="`)
+* {applies_to}`stack: ga 9.1.4` The unencoded `id:api_key` format (`api_key: "foo:bar"`) or the Base64-encoded `id:api_key` format (`api_key: "Zm9vOmJhcgo="`)
+* All earlier releases can only use the unencoded `id:api_key` format (`api_key: "foo:bar"`)
```

Comment:
It looks like the `9.2.0` and 9.1.4` lines are the same. Is that intentional? If it is, you only need to specify that it was added in 9.1.4 and the tooltip will clarify that it's 9.1.4 and later. 

```suggestion
* {applies_to}`stack: ga 9.1.4` The unencoded `id:api_key` format (`api_key: "foo:bar"`) or the Base64-encoded `id:api_key` format (`api_key: "Zm9vOmJhcgo="`)
* All earlier releases can only use the unencoded `id:api_key` format (`api_key: "foo:bar"`)
```

Reply:
Oh! Thank you. That was very confusing for me.